### PR TITLE
Improved output

### PR DESCRIPTION
**Pull request by paulino1997**
I added one simple output

### CODE REVIEW

File: Examples/Ex00_properties.m [hunk 1]
```diff
@@ -2,6 +2,7 @@
 %* HGS 2.0 (adapted from the original HGS 1.3) 
 %* Original by Arnau Miro, Pau Manent, Manel Soria 
 %* Adapted by Caleb Fuster
+%* Commented by Paulino Gil -> Only this comment
```
Comment (ManelSoria) [inline Examples/Ex00_properties.m]: Paulino, this is the hall of fame. You should contribute a more significant change to be there.
But don't be disappointed and keep trying.

File: Examples/Ex00_properties.m [hunk 2]
```diff
@@ -15,7 +16,10 @@
 
 % Compute the propeties of a mixture of 1 mol of N2, 2 mol of CH4 and 3 mol
 % of C3H8 at 3 bar and 400K 
-[Mm,Cp,Cv,H,S,G,Rg,gamma,a]=HGSprop({'N2','CH4','C3H8'},[1,2,3],400,3)
+[Mm,Cp,Cv,H,S,G,Rg,gamma,a]=HGSprop({'N2','CH4','C3H8'},[1,2,3],400,3); 
+fprintf('propeties of a mixture of 1 mol of N2 , 2 mol of CH4 and 3 mol of C3H8 at 3 bar and 400K \n');
+fprintf('Mm = %f, Cp = %f, Cv = %f \n',Mm,Cp,Cv);
+
```
Comment (ManelSoria) [inline Examples/Ex00_properties.m:21]: ok

**Review by ManelSoria — CHANGES_REQUESTED** [review]

File: Examples/Ex00_properties.m (hunk 1 above)
Comment (paulino1997) [reply Examples/Ex00_properties.m]: Sure ! You're right...I hope I will not disappoint you.

**Review by ManelSoria — APPROVED** [review]
ok !!

**Pull request merged**